Example/Tests/GalleryCoordinatorTests.swift

```diff
@@ -12,6 +12,7 @@ import XCTest
 final class GalleryCoordinatorTests: XCTestCase {
     
     let galleryManager = GalleryManagerMock()
+    let giniConfiguration: GiniConfiguration = GiniConfiguration.sharedConfiguration
```

Comment:
You keep using the shared configuration in tests, but you leave it in undefined state after the test is run. It can lead to subtle bug, especially since the order of tests is not guaranteed.

I'd suggest not using the shared configuration. Or at the very least, reset the values in the `tearDown`.